features/step_definitions/employers_steps.rb

```diff
@@ -442,7 +442,8 @@
 end
 
 When(/^.+ go(?:es)? to the benefits tab$/) do
-  find(".interaction-click-control-benefits").click
+  find(EmployerHomePage.benefits_link).click
+  sleep 5
```

Comment:
Let's not do sleep 5, we should instead add `wait: 5` to the next `find` so that it will only wait as long as needed.